fairlearn/metrics/_disaggregated_result.py

```diff
@@ -172,7 +172,7 @@ def apply_grouping(
             elif errors == "coerce":
                 # Fill all impossible columns with NaN before grouping metric frame
                 mf = self.by_group.copy()
-                mf = mf.map(lambda x: x if np.isscalar(x) else np.nan)
+                mf = mf.applymap(lambda x: x if np.isscalar(x) else np.nan)
```

Comment:
we're not testing these?

Reply:
According to CodeCov we are:
https://app.codecov.io/gh/fairlearn/fairlearn/pull/1298?src=pr&el=h1&utm_medium=referral&utm_source=github&utm_content=checks&utm_campaign=pr+comments&utm_term=fairlearn